src/patterns/dosdp-patterns/bone_length_from_to.yaml

```diff
@@ -0,0 +1,77 @@
+---
+pattern_name: bone_length_from_to
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/bone_length_from_to.yaml
+
+description: 'A bone length trait that is the distance between two anatomical reference points measured in a straight line.'
+
+#  examples:
+#    - http://purl.obolibrary.org/obo/FOVT_0001070  # femur length from caput to lateral condyle
+#    - http://purl.obolibrary.org/obo/FOVT_0001077  # length of talus along the midline
+#    - http://purl.obolibrary.org/obo/FOVT_0000007  # femur length from caput to medial condyle
+
+contributors:
+  - https://orcid.org/0000-0002-7073-9172  # David Osumi-Sutherland
+  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
+  - https://orcid.org/0000-0002-7356-1779  # Nicolas Matentzoglu
+  - https://orcid.org/0000-0003-2699-3066  # Meghan A. Balk
+
+classes:
+  skeletal_element: UBERON:0004765
+  anatomical_entity: UBERON:0001062
+  length: PATO:0000122
+  line: BFO:0000026  # one-dimensional spatial region
+  physical_object_quality: PATO:0001241
+
+relations:
+  connected_to: RO:0002170
+  characteristic_of: RO:0000052
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+  xref: oio:hasDbXref
+
+vars:
+  bone: "'skeletal_element'"
+  from: "'anatomical_entity'"
+  to: "'anatomical_entity'"
+  fovt: "'physical_object_quality'"
+
+name:
+  text: "%s length from %s to %s"
+  vars:
+    - bone
+    - from
+    - to
+
+annotations:
+  - annotationProperty: related_synonym
+    text: "%s length"
+    vars:
+      - bone
+
```

Comment:
add annotation: "measured_in_taxon" (value NCBI taxon) and "not_measured_in_taxon" (value NCBI taxon)

Reply:
Can you explain a bit more about these?  Measured by who?  If it is not been measured in taxon X at time T, might it be measure later so this goes out of date?  Perhaps the intent is to have a list of taxonomic groupings where this applies and groupings where it doesn't.  In that case, maybe regular taxon restrictions would be better?

Reply:
Its documentative information (metadata), nothing logical. @megbalk wants to capture that for a species a trait is typically measured - while for others, even if the trait certainly exists, it is not typically the subject of a measurement. These were originally taxon constraints, but I advised that this is way too strong a statement.